index.ts

```diff
@@ -0,0 +1,5 @@
+import zumokit from './src/ZumoKit';
+
+export default zumokit;
+export * from './src/types/exported';
+export * from './src/interfaces';
```

Comment:
zumokit singleton is default export, while interfaces and types are exported as names exports.